metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -372,7 +372,19 @@ def deploy(
         """
     )
 
-    response = datahub_graph.execute_graphql(graphql_query, variables=variables)
+    try:
+        response = datahub_graph.execute_graphql(
+            graphql_query, variables=variables, format_exception=False
+        )
+    except GraphError as graph_error:
+        try:
+            error = json.loads(str(graph_error).replace('"', '\\"').replace("'", '"'))
```

Comment:
Parsing graphql errors seems super annoying... this is just to display an error to the user and has a blanket try / catch so think this is safe to put in